### ISSUE TITLE
BUG : amazon.titan-text-express-v1 is not yet implemented as a feedback provider

### ISSUE THREAD

**Issue by SanjuMLGeek** [truera/trulens#886]
Hello,

I have below simple code which failed with error `bedrock request failed <class 'NotImplementedError'>=The model selected, amazon.titan-text-express-v1, is not yet implemented as a feedback provider`.

As you can see in commented code, I tried anthropic.claude-v2 and other Bedrock models by cross referencing model_id from this document : https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html

But it fails for all models. I have Bedrock model enabled and it works when I independently invoke them as provided in screenshot. 

```
# Initialize provider class
tru = Tru()
tru.reset_database()

#ftitan = fBedrock(model_id = "anthropic.claude-v2")
ftitan = fBedrock()

# Define a groundedness feedback function
f_comprehensiveness = (
    Feedback(ftitan.comprehensiveness_with_cot_reasons, name = "Comprehensiveness")
    .on_input_output()
)

tru_recorder = TruChain(
    llm_chain_summary,
    app_id='Exp_1_Claim_Summary',
    feedbacks=[f_comprehensiveness]
)

with tru_recorder as recording:
    completion = llm_chain_summary({"context": notes}, return_only_outputs=True)
```
Error Screenshot: The multiple error tries comes from my multiple attemps by changing model_id. 

<img width="1239" alt="image" src="https://github.com/truera/trulens/assets/43816307/bcd48abb-40cf-43c2-ac7e-aea049f4cd92">

Working Code using Bedrock through Langchain:

<img width="1239" alt="image" src="https://github.com/truera/trulens/assets/43816307/0099ac4c-c760-453d-a02b-9c80ecf84acc">

**Comment by joshreini1** [comment]
Thanks @SanjuMLGeek - fixed in this [PR](https://github.com/truera/trulens/pull/887). Can you install from that branch and give it a try?

The only model available in Bedrock not yet implemented in TruLens should be Llama-2

**Comment by SanjuMLGeek** [comment]
Hi @joshreini1 , I used the branch and commit for pip install and it fails with below error message. However, I provided only 1 error but all 3 attempts with different ways of pip install failed. 

!pip install git+https://github.com/truera/trulens.git@josh/bedrock_fix
!pip install git+https://github.com/truera/trulens.git@f882cae4a911a89c00b6f713b368adc20fa1ee87
!pip install git+https://github.com/truera/trulens.git@177a6c77d5235b9c39f37a29cca558a7a6fd9148

```
Collecting git+https://github.com/truera/trulens.git@177a6c77d5235b9c39f37a29cca558a7a6fd9148
  Cloning https://github.com/truera/trulens.git (to revision 177a6c77d5235b9c39f37a29cca558a7a6fd9148) to /tmp/pip-req-build-k2_9v4g4
  Running command git clone --filter=blob:none --quiet https://github.com/truera/trulens.git /tmp/pip-req-build-k2_9v4g4
  Running command git rev-parse -q --verify 'sha^177a6c77d5235b9c39f37a29cca558a7a6fd9148'
  Running command git fetch -q https://github.com/truera/trulens.git 177a6c77d5235b9c39f37a29cca558a7a6fd9148
  Running command git checkout -q 177a6c77d5235b9c39f37a29cca558a7a6fd9148
  Resolved https://github.com/truera/trulens.git to commit 177a6c77d5235b9c39f37a29cca558a7a6fd9148
ERROR: git+https://github.com/truera/trulens.git@177a6c77d5235b9c39f37a29cca558a7a6fd9148 does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
```

**Comment by joshreini1** [comment]
Hey @SanjuMLGeek - you're just missing the subdirectory. Try:

`pip install git+https://github.com/truera/trulens@josh/bedrock_fix#subdirectory=trulens_eval`

**Comment by joshreini1** [comment]
Hey @SanjuMLGeek - this change is released now in 0.22.2:

https://github.com/truera/trulens/releases/tag/trulens-eval-0.22.2

**Comment by SanjuMLGeek** [comment]
@joshreini1 - It solved the problem. Thank you.
